sklearn/ensemble/_bagging.py

```diff
@@ -237,6 +239,26 @@ class BaseBagging(BaseEnsemble, metaclass=ABCMeta):
     instead.
     """
 
+    _parameter_constraints = {
+        "base_estimator": "no_validation",
+        "n_estimators": [Interval(Integral, 1, None, closed="left")],
```

Comment:
these could be in BaseEnsemble but let's do that once all the child classes of BaseEnsemble are done since we've already merged some of them

Reply:
Ok great